src/main/java/com/hazelcast/web/HazelcastInstanceLoader.java

```diff
@@ -100,32 +130,26 @@ private static Config getServerConfig(String mapName, URL configUrl, String sess
                 throw new ServletException(e);
             }
         }
-        if (sessionTTLConfig == null) {
-            sessionTTLConfig = SESSION_TTL_DEFAULT_SECONDS;
+
+        if (!isNullOrEmptyAfterTrim(instanceName)) {
```

Comment:
@mesutcelik, correct me if I'm wrong but I remember that after further discussion we had concluded that if there's no instance with that name, we should create one with that name because it will be more convenient for the user. The alternative (making the user create Hazelcast instance himself) is conceptually clearer but more inconvenient for the user.